**Exercise:** In the BRFSS (see Section 5.4), the distribution of heights is roughly normal with parameters µ = 178 cm and σ = 7.7 cm for men, and µ = 163 cm and σ = 7.3 cm for women.

In order to join Blue Man Group, you have to be male between 5’10” and 6’1” (see http://bluemancasting.com). What percentage of the U.S. male population is in this range? Hint: use `scipy.stats.norm.cdf`.

In [10]:
import scipy.stats
import brfss
import analytic
import thinkstats2
import thinkplot

In [18]:
# Calculate ft/in -> cm
avg_ht = (5*12 + 10) * 2.54
max_ht = (6*12 + 1) * 2.54

In [19]:
avg_ht
# In the prompt he rounds to 178, so I will round to that as well

177.8

In [20]:
max_ht

185.42000000000002

In [24]:
# I calculate the percentile rank of the max height, and then subtract the bottom half of the distribution,
# since the 5'10" mark is the median height in a normal distribution of height.

scipy.stats.norm.cdf(max_ht, loc = 178, scale = 7.7) - 0.5

# The result is the percentage of male heights that fall within the 5'10" to 6'1" range.

0.3323858654963072

**Answer:** Approximately 33.2% of men are in the range of eligibility for the Blue Man Group